icdiff

```diff
@@ -601,17 +601,18 @@ def set_cols_option(options):
 def validate_has_two_arguments(parser, args):
     if len(args) != 2:
         parser.print_help()
-        sys.exit()
+        sys.exit(2)
```

Comment:
How about defining constants?  Something like:

     EXIT_NO_DELTA 0
     EXIT_DIFFS_FOUND 1
     EXIT_INVOCATION_INCORRECT 2

Reply:
Done, should be resolved in commit [39957b0](https://github.com/jeffkaufman/icdiff/pull/195/commits/39957b07a26f1717fc8406f85fc0105d7224f686)